sql/7_0_2-to-7_0_3_upgrade.sql

```diff
@@ -113,3 +113,7 @@
 ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
 #EndIf
 
+
+#IfMissingColumn layout_group_properties grpsec_copy_allow
+ALTER TABLE `layout_group_properties` ADD `grpsec_copy_allow` tinyint(1) DEFAULT 0 AFTER `grp_last_update`;
```

Comment:
remove the `AFTER grp_last_update`
(just potential for breaking in upgrade in future if for some reason that column is removed)

Reply:
also would make it identical to the type you have in database.sql to ensure no differences in upgraded versions. ie. `not null default 0`